Packs/CortexResponseAndRemediation/ReleaseNotes/1_1_77.md

```diff
@@ -0,0 +1,40 @@
+
+#### Playbooks
+
+##### New: Script file added to startup-related Registry keys
+
+This playbook is designed to handle "Script file added to startup-related Registry keys" alerts.
+
+##### Playbook Stages:
+
+Analysis:
+
+- Extract the script path from the registry key value.
+- Check the reputation of the associated processes.
+
+###### Investigation:
+
+- Searches for related XSIAM alerts to identify potential attack patterns.
+- Retrieves the script file and relevant details for investigation.
+- Analyzes the script for malicious or suspicious parameters:
+  - If malicious parameters are found, proceed to remediation.
+  - If suspicious parameters are found, detonate and analyze the script using WildFire Sandbox.
+- Checks the script file reputation.
+- Detonates and analyzes the script using WildFire Sandbox.
+
+Remediation:
+
+- Terminates the malicious process.
+- Quarantines the malicious process.
+- Adds the malicious process hash to the blocklist.
+- Calculates the script file’s SHA256 hash if not already available.
+- Deletes the Registry key value (Requires analyst approval).
+- Adds the script file hash to the blocklist (Requires analyst approval).
+- Automatically Close the alert.
+
+
+#### Triggers Recommendations
+
+##### New: Script file added to startup-related Registry keys
+
+- New: Added a new triggers recommendation- Script file added to startup-related Registry keys that This trigger is responsible for handling the 'Script file added to startup-related Registry keys' alerts.
```

Comment:
Added a new trigger to execute **Script file added to startup-related Registry keys** alerts.